### PR TITLE
games-strategy/ufo-ai: version bump to 2.5

### PR DESCRIPTION
**Pull request by tomboy-64**
This is a long-standing version bump from [Bug #515660](https://bugs.gentoo.org/show_bug.cgi?id=515660). Includes changes by L29Ah to make it work with hardened.

### CODE REVIEW

File: games-strategy/ufo-ai/ufo-ai-2.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,130 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=3
```
Comment (hasufell) [inline games-strategy/ufo-ai/ufo-ai-2.5.ebuild]: bump to EAPI 5 and test the package with it

File: games-strategy/ufo-ai/ufo-ai-2.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+DESCRIPTION="UFO: Alien Invasion - X-COM inspired strategy game"
+HOMEPAGE="http://ufoai.sourceforge.net/"
+SRC_URI="mirror://sourceforge/ufoai/${MY_P}-source.tar.bz2
+	mirror://sourceforge/ufoai/${MY_P}-data.tar"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="debug server editor +client sse profile"
```
Comment (hasufell) [inline games-strategy/ufo-ai/ufo-ai-2.5.ebuild]: sort USE flags alphabetically

File: games-strategy/ufo-ai/ufo-ai-2.5.ebuild [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+# Dependencies and more instructions can be found here:
+# http://ufoai.ninex.info/wiki/index.php/Compile_for_Linux
+DEPEND="!server? (
```
Comment (hasufell) [inline games-strategy/ufo-ai/ufo-ai-2.5.ebuild]: sort dependencies alphabetically and fix indenting

File: games-strategy/ufo-ai/ufo-ai-2.5.ebuild [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+		virtual/opengl
+		 virtual/glu
+		media-libs/libsdl2
+		media-libs/sdl2-image[jpeg,png]
+		media-libs/sdl2-ttf
+		media-libs/sdl2-mixer
+		virtual/jpeg
```
Comment (hasufell) [inline games-strategy/ufo-ai/ufo-ai-2.5.ebuild]: missing slot on `virtual/jpeg`

File: games-strategy/ufo-ai/ufo-ai-2.5.ebuild [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+		media-libs/libogg
+		media-libs/libvorbis
+		x11-proto/xf86vidmodeproto
+	)
+	net-misc/curl
+	sys-devel/gettext
+	sys-libs/zlib
+	editor? (
+		dev-libs/libxml2
+		virtual/jpeg
```
Comment (hasufell) [inline games-strategy/ufo-ai/ufo-ai-2.5.ebuild]: missing slot on `virtual/jpeg`

File: games-strategy/ufo-ai/ufo-ai-2.5.ebuild [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+pkg_setup() {
+	if use profile; then
+		ewarn "USE=\"profile\" is incompatible with the hardened profile's -pie flag."
+	fi
+}
+
+src_unpack() {
+	unpack ${MY_P}-source.tar.bz2
+	cd "${S}"
+	pwd
```
Comment (hasufell) [inline games-strategy/ufo-ai/ufo-ai-2.5.ebuild]: ???

File: games-strategy/ufo-ai/ufo-ai-2.5.ebuild [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+
+pkg_setup() {
+	if use profile; then
+		ewarn "USE=\"profile\" is incompatible with the hardened profile's -pie flag."
+	fi
+}
+
+src_unpack() {
+	unpack ${MY_P}-source.tar.bz2
+	cd "${S}"
```
Comment (hasufell) [inline games-strategy/ufo-ai/ufo-ai-2.5.ebuild]: missing `|| die`

File: games-strategy/ufo-ai/ufo-ai-2.5.ebuild [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+		--datadir="${GAMES_DATADIR}/${PN/-}"
+		--localedir="${EPREFIX}/usr/share/locale/"
+		--prefix="${GAMES_PREFIX}"
+	"
+	echo "./configure ${myconf}"
+	./configure ${myconf} || die
+}
+
+src_compile() {
+	echo "Running emake!"
```
Comment (hasufell) [inline games-strategy/ufo-ai/ufo-ai-2.5.ebuild]: useless echo

File: games-strategy/ufo-ai/ufo-ai-2.5.ebuild [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+	emake || die
+	echo "Running emake lang!"
```
Comment (hasufell) [inline games-strategy/ufo-ai/ufo-ai-2.5.ebuild]: useless echo

File: games-strategy/ufo-ai/ufo-ai-2.5.ebuild [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="debug server editor +client sse profile"
+
+# Dependencies and more instructions can be found here:
+# http://ufoai.ninex.info/wiki/index.php/Compile_for_Linux
+DEPEND="!server? (
+		virtual/opengl
+		 virtual/glu
+		media-libs/libsdl2
```
Comment (hasufell) [inline games-strategy/ufo-ai/ufo-ai-2.5.ebuild]: are there _no_ USE flags at all necessary on libsdl2? as in... if you compile `media-libs/libsdl2` with __all__ USE flags disabled, does ufo-ai still work?

File: games-strategy/ufo-ai/ufo-ai-2.5.ebuild (hunk 2 above)
Comment (hasufell) [inline games-strategy/ufo-ai/ufo-ai-2.5.ebuild]: `client` is not games standard. games in gentoo use the `dedicated` USE flag

**Comment by tomboy-64** [comment]
Closing this request for now, as it needs much more work, it seems. Cleaning out the deps and figuring out useflags in DEPEND also broke compilation.

**Pull request closed without merging**